plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlJdbcConfig.java

```diff
@@ -42,8 +40,7 @@ public boolean isUrlWithoutDatabase()
     {
         try {
             Driver driver = new Driver();
-            Properties properties = driver.parseURL(getConnectionUrl(), null);
-            return (properties == null) || (driver.database(properties) == null);
+            return driver.acceptsURL(getConnectionUrl());
```

Comment:
Would this test https://github.com/trinodb/trino/pull/8290 still pass after the change here?

Reply:
No.

Reply:
```
assertFalse(isUrlWithoutDatabase("jdbc:mysql://example.net:3306/somedatabase"));   // fails
```

Reply:
> Would this test #8290 still pass after the change here?

@findepi It will work now with  https://github.com/trinodb/trino/pull/8060/commits/2eb59ae2265e0e9f53daafb6b2cd5410b53814cc